packages/iframe-channel-provider/src/channel-provider.ts

```diff
@@ -182,6 +182,9 @@ export class IFrameChannelProvider implements IFrameChannelProviderInterface {
   off: OffType = (method, params) => this.events.off(method, params);
 
   protected async onMessage(event: MessageEvent) {
+    if (!event.data.jsonrpc) {
+      return;
+    }
     const message = parseNotification(event.data); // Narrows type, throws if it does not fit the schema
```

Comment:
We don't want to reach L188 if the message is not json-rpc, since it might throw an error when trying to parse the message as a notification. 

Non json-rpc messages included some of our logging, and the `Wallet Ready` message.